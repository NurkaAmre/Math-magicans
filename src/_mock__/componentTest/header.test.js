import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../../component/header';

describe('Header', () => {
  test('should render header', () => {
    const { container } = render(<Router><NavBar /></Router>);
    expect(container).toMatchSnapshot();
  });
});
