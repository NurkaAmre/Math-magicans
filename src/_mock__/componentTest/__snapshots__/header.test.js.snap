// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header 1`] = `
<div>
  <header
    class="top-section"
  >
    <h2>
      Math Magicians
    </h2>
    <nav
      class="navigation"
    >
      <a
        aria-current="page"
        class="link active"
        href="/"
      >
        Home
      </a>
      <a
        class="link"
        href="/calculator"
      >
        Calculator
      </a>
      <a
        class="link"
        href="/quotes"
      >
        Quotes
      </a>
    </nav>
  </header>
</div>
`;

exports[`NavBar should render 1`] = `
<div>
  <header
    class="top-section"
  >
    <h2>
      Math Magicians
    </h2>
    <nav
      class="navigation"
    >
      <a
        aria-current="page"
        class="link active"
        href="/"
      >
        Home
      </a>
      <a
        class="link"
        href="/calculator"
      >
        Calculator
      </a>
      <a
        class="link"
        href="/quotes"
      >
        Quotes
      </a>
    </nav>
  </header>
</div>
`;
